pkgs/tools/misc/iceportal/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
```

Comment:
Please use buildGoModule so that the package is not added to the deprecation pile immediately.